include/deal.II/multigrid/multigrid.templates.h

```diff
@@ -271,25 +268,24 @@ Multigrid<VectorType>::level_step(const unsigned int level,
 
   transfer->restrict_and_add (level, defect2[level-1], t[level]);
 
-  // Every cycle needs one recursion, the V-cycle, which is included here for
-  // the sake of the F-cycle, needs only one. As opposed to the V-cycle above
-  // where we do not need to set the solution vector to zero (as that has been
-  // done in the init functions), we also need to set the content in the
-  // solution from previous cycles to zero.
+  // Every cycle starts with a recursion of its type. As opposed to the
+  // V-cycle above where we do not need to set the solution vector to zero (as
+  // that has been done in the init functions), we need to zero out the values
+  // from previous cycles.
   solution[level-1] = 0;
   level_step(level-1, cycle);
 
   // For W and F-cycle, repeat the process on the next coarser level except
   // for the coarse solver which we invoke just once
   if (level>minlevel+1)
     {
-      // while the W-cycle repeats itself
+      // while the W-cycle repeats itself,
```

Comment:
if you intend to run the comment off this line and into the next comment, maybe do
```
  // while the W-cycle repeats itself, ...
  code;
  // ...the F-cycle does a V-Cycle after an F-Cycle, ...
  code;
  // ...and the V-Cycle does nothing.
```